package.json

```diff
@@ -27,7 +27,8 @@
     ]
   },
   "devDependencies": {
-    "beachball": "^1.13.4",
+    "beachball": "^1.32.0",
+    "husky": "^4.2.5",
```

Comment:
>husky [](start = 5, length = 5)

Is this intentional? I didn't see a `.husky*` file or husky settings in package.json...

Reply:
@NickGerleman : Approving, but please take a look at this comment from Danny

Reply:
@dannyvv the story for Husky is that we used to have it installed, then we removed it, but it left a pre-commit hook on everyone's machine to run it so soemone added it back. Master has Husky, so I went ahead and kept it here when resolving merge conflicts.